token/program-2022/src/extension/confidential_transfer_fee/processor.rs

```diff
@@ -367,41 +470,35 @@ pub(crate) fn process_instruction(
             )
         }
         ConfidentialTransferFeeInstruction::WithdrawWithheldTokensFromMint => {
-            msg!("ConfidentialTransferInstruction::WithdrawWithheldTokensFromMint");
-            #[cfg(all(feature = "zk-ops", feature = "proof-program"))]
+            msg!("ConfidentialTransferFeeInstruction::WithdrawWithheldTokensFromMint");
+            #[cfg(feature = "zk-ops")]
             {
                 let data = decode_instruction_data::<WithdrawWithheldTokensFromMintData>(input)?;
-                return process_withdraw_withheld_tokens_from_mint(
+                process_withdraw_withheld_tokens_from_mint(
                     program_id,
                     accounts,
+                    &data.new_decryptable_available_balance,
                     data.proof_instruction_offset as i64,
-                );
-            }
-            #[cfg(not(all(feature = "zk-ops", feature = "proof_program")))]
-            {
-                Err(ProgramError::InvalidInstructionData)
+                )
             }
         }
         ConfidentialTransferFeeInstruction::WithdrawWithheldTokensFromAccounts => {
-            msg!("ConfidentialTransferInstruction::WithdrawWithheldTokensFromAccounts");
-            #[cfg(all(feature = "zk-ops", feature = "proof-program"))]
+            msg!("ConfidentialTransferFeeInstruction::WithdrawWithheldTokensFromAccounts");
+            #[cfg(feature = "zk-ops")]
             {
                 let data =
                     decode_instruction_data::<WithdrawWithheldTokensFromAccountsData>(input)?;
-                return process_withdraw_withheld_tokens_from_accounts(
+                process_withdraw_withheld_tokens_from_accounts(
                     program_id,
                     accounts,
                     data.num_token_accounts,
+                    &data.new_decryptable_available_balance,
                     data.proof_instruction_offset as i64,
-                );
-            }
-            #[cfg(not(all(feature = "zk-ops", feature = "proof_program")))]
-            {
-                Err(ProgramError::InvalidInstructionData)
+                )
             }
```

Comment:
You'll also need the Error variant if the feature is off

Reply:
Oh sorry I missed it!